md/runtime-monitoring.md

```diff
@@ -0,0 +1,102 @@
+* The number of currently running connector works, under the logical key name **org.bonitasoft.engine.connector.connectors.running**
+* The number of currently pending connector works, waiting in the connector work queue to be treated,
+under the logical key name **org.bonitasoft.engine.connector.connectors.pending**
+* The total number of executed connector works (since the last start of Bonita platform), under the logical key name **org.bonitasoft.engine.connector.connectors.executed**
+
+
+
+## Activating specific monitoring metrics
+
+Retrieve current configuration by running:
+```bash
+./setup/setup.sh pull
+```
+and edit file `./setup/platform_conf/current/platform_engine/bonita-platform-community-custom.properties`.  
+You will see, in the `# Monitoring` section, two series of properties with their default value:
+
+    ## Monitoring
+    ## publish metrics to JMX:
+    #org.bonitasoft.engine.monitoring.jmx.enable=true
+    ## periodically print metrics to logs (bonita-related metrics only):
+    #org.bonitasoft.engine.monitoring.logging.enable=false
+    ## print to logs every minute by default (in the ISO-8601 duration format):
+    #org.bonitasoft.engine.monitoring.logging.step=PT1M
+    #
+    ## publish metrics related to JVM memory:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.memory.enable=false
+    ## publish metrics related to JVM Threads:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.threads.enable=false
+    ## publish metrics related to JVM garbage collection:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.gc.enable=false
+    ## publish technical metrics related to Worker / Connector thread pools:
+    #org.bonitasoft.engine.monitoring.metrics.executors.enable=false
+
+The first series is the list of **publishers** that can be activated.  
+In Community edition, 2 publishers are provided:
+* JMX (enabled by default), that allows to use any JMX console to monitor your favorite metrics (except JVM metrics)
+* Logging (disabled by default), that prints to standard Bonita log file the Bonita-related metrics. Print interval can
+be changed (property `org.bonitasoft.engine.monitoring.logging.step`).
+
+The second series is the list of **metrics** (counters) that can be exposed.  
+All configurable metrics are disabled by default and can be enabled separately.  
+They provide information about:
+* the running JVM memory
+* JVM threads
+* Garbage collection usage
+* Worker / Connector thread pools
+
+Each of these metrics provides many different counters to finely understand what is going on.
+
+To change any value, **uncomment the line by removing the # character**, and change the true / false value.  
+Server restart is required for the changes to take effect.
+
+## Subscription-only monitoring publisher
+
+Additionally, Bonita Subscription editions can publish to a REST endpoint in the
```

Comment:
I've been told a couple of time that we are supposed to use the word "Enterprise" now, even if it doesn't reflect the whole truth

Reply:
ah?

Reply:
🙈

Reply:
ask your PO for that. All pages still reference all 'subscription' editions
For instance: https://documentation.bonitasoft.com/bonita/7.9/environments
```
Note: For Enterprise, Performance, Efficiency, and Teamwork editions only.
```

Reply:
discussed and resolved orally with @NathalieC . Closing this comment.